### DISCUSSION TITLE
Need help diagnosing 403 error at Admin

### DISCUSSION THREAD

**Discussion by orawalters** [POPWorldMedia/POPForums#365] [General]
Hello,
I need help accessing the PopForums Admin page in my development environment. So far I've tried relaunching the server and clearing my cookies. Regardless, after I sign on clicking on the link to PopForums admin gives me a blank 403 page.

**Comment by jeffputz** [comment]
This is not an issue, it's a user problem. Moving to discussions.

If it's coming up 403, the user that you're signing in as is not an admin user.

**Reply by orawalters** [reply]
@jeffputz This is the admin user created by the setup.

**Reply by orawalters** [reply]
@jeffputz Just checked the database to be sure, UserID 1 has both Admin and Moderator roles.

**Reply by jeffputz** [reply]
@orawalters that's the correct state. Have you tried to debug looking at `PopForumsAuthorizationMiddleware`? You may have something out of order in your `Program` setup (see the included code or sample project).

**Reply by orawalters** [reply]
@jeffputz It turned out I messed up the routing. Sorry to bother you.
